app/assets/stylesheets/application/static.sass

```diff
@@ -19,3 +19,30 @@ body[data-controller="static"]
     margin-right: 8px
     height: 128px
     width: 128px
+  hr
+    border-color: $braveLightOrange
+  .form-control
+    background: $bravePurple
+    color: white
+  .control-label
+    width: 100%
+    color: white
+  .btn
```

Comment:
Once we standardize colors, I think it's important to move button styles into the shared button styles file to ensure we're consistent across the app. cc @alexwykoff